Sources/KeyboardShortcuts/Localization/ja.lproj/Localizable.strings

```diff
@@ -0,0 +1,5 @@
+"record_shortcut" = "キーを記録";
+"press_shortcut" = "キーを押してください";
+"keyboard_shortcut_used_by_menu_item" = "このショートカットは既に“%@”で使われており使えません。 ";
```

Comment:
```suggestion
"keyboard_shortcut_used_by_menu_item" = "このショートカットは既に“%@”で使われており使えません。";
```

Should there really be a space at the end?

Reply:
Maybe it's possible to remove it?